charts/capsule/templates/deployment.yaml

```diff
@@ -37,7 +37,15 @@ spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       {{- end }}
+      {{- if .Values.manager.hostPID }}
+      hostPID: {{ .Values.manager.hostPID }}
+      {{- else }}
+      hostPID: false
+      {{- end }}
+      {{- if eq .Values.priorityClassName "" }}
+      {{- else }}
```

Comment:
?

Reply:
Maybe I might remove it. Was probably a typo.